project.clj

```diff
@@ -1,4 +1,4 @@
-(defproject me.dryewo/clj-nakadi-java "0.0.2"
+(defproject me.dryewo/clj-nakadi-java "0.0.3"
```

Comment:
Please keep the version number the same, it will be bumped as part of the release procedure.